SPECS/nano/nano.spec

```diff
@@ -1,28 +1,30 @@
-Summary:	Text editor
-Name:		nano
-Version:	3.0
-Release:        3%{?dist}
-License:	GPLv3+
-URL:		http://www.nano-editor.org/
-Group:		Applications/Editors
-Source0:	http://www.nano-editor.org/dist/v3/%{name}-%{version}.tar.xz
-%define sha1 nano=0396d85df64a7927ca2d3425a0263dacf8502220
+Summary:        Text editor
+Name:           nano
+Version:        6.0
+Release:        1%{?dist}
+License:        GPLv3+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-BuildRequires:	ncurses-devel
-Requires:	ncurses
+Group:          Applications/Editors
+URL:            https://www.nano-editor.org/
+Source0:        http://www.nano-editor.org/dist/v6/%{name}-%{version}.tar.xz
+%define sha1 nano=0396d85df64a7927ca2d3425a0263dacf8502220
+BuildRequires:  ncurses-devel
+Requires:       ncurses
 
 %description
 The Nano package contains a small, simple text editor
 
 %package lang
-Summary:	Lang for nano
-Requires:	%{name} = %{version}
+Summary:        Lang for nano
+Requires:       %{name} = %{version}
```

Comment:
We should tighten up all of these runtime requirements to `%{version}-%{release}`, so we can keep any changed subpackages synced with each other on users' machines.